silver/install-guide/index.md

```diff
@@ -10,23 +10,30 @@ menu_weight: 10.0
 
 ## Prerequisites
 
-[Java JDK, version 1.7](http://www.oracle.com/technetwork/java/javase/downloads/index.html) and [Apache ANT](http://ant.apache.org/bindownload.cgi). For Ubuntu users:
+[Java JDK, version 1.7](http://www.oracle.com/technetwork/java/javase/downloads/index.html), [Apache ANT](http://ant.apache.org/bindownload.cgi), Git and wget. For Ubuntu users:
 
 ```
-apt-get install default-jdk-headless ant
+apt-get install default-jdk ant git wget
 ```
 
 For OSX, using Homebrew (install a JDK separately):
 
 ```
-brew install coreutils ant
+brew install coreutils ant git wget
 ```
 
 ## Getting Silver
 
-The **latest release** can be [downloaded here](/downloads/).
+To clone from GitHub,
+wherever you wish to checkout the repository, run:
+
+```
+$ git clone https://github.com/melt-umn/silver.git
+$ cd silver
+silver$ ./update
```

Comment:
Following this, there should be some explanation of what update does, so they understand why they're doing it.